librust/proc_macro/rust/bridge/literal.rs

```diff
@@ -0,0 +1,212 @@
+use bridge::span::Span;
+use std::convert::TryInto;
+use std::ffi::c_uchar;
+
+extern "C" {
+    fn Literal__drop(literal: *const Literal);
+    fn Literal__string(str: *const c_uchar) -> Literal;
+    fn Literal__byte_string(bytes: *const u8) -> Literal;
+}
+
+#[repr(C)]
+#[derive(Clone)]
+pub enum Unsigned {
+    Unsigned8(u8),
+    Unsigned16(u16),
+    Unsigned32(u32),
+    Unsigned64(u64),
+    // FIXME: 128 bits ffi is not safe for now
+    // https://github.com/rust-lang/rust/issues/54341
+    //
+    // Unsigned128(u128),
```

Comment:
`Unsigned128(u64, u64)` would work, right?

Reply:
Yes, this is what I had in my mind.